pkg/pool-manager/pool.go

```diff
@@ -147,6 +150,15 @@ func (p *PoolManager) InitMaps() error {
 	return nil
 }
 
+func (p *PoolManager) IsMacInRange(macAddress string) bool {
+	macAddressHW, _ := net.ParseMAC(macAddress)
```

Comment:
I think its better to keep the parsing outside of this kind of function, what you think?

Reply:
I agree. it's redundant. will remove.

Reply:
Since the inRange() is using byte[] type MAC Address, I need to translate the mac address, which is saved as sting on the cache maps.

since it is an internal matter, I think it's better to keep it here.